### DISCUSSION TITLE
Discord community call: 26 October 2024

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#7118] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: October 26th, 2024 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/oct-26-2024/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- Joost's update (posted below)
- Update on project to port designs docs to Docusaurus. https://github.com/freesewing/freesewing/issues/7168

**Comment by joostdecock** [comment]
Hey there,

Unfortunately, I won't be able to attend today's call because it's my mon's 80th birthday party and I doubt I can use this call as a valid excuse.

That being said, I wanted to share some stuff that's been going on recently so that everyone can read up about it:

- Progress has been made on porting the various designs to docusaurus. I want to thank all those who stepped up to help out, I really appreciate it.
- We'll continue preparing the docs site until we've got it to be the way we want it to be
- In parallel, we've made the decision to not try and shoehorn all of these recent changes into some 3.x release, and instead focus our attention on a 4.0 release. The driving factor here is the fact that NodeJS 18 is out of active support. See: #7204
- I hope to be able to find more time for all this in the last couple of months of the year, and I'd love to try and get v4 out the door in 2024. That being said, it will be ready when it's ready.
- The release cycle of NodeJS releases means that we are typically going to have a major release every year. So that's perhaps also something we should say out loud rather than just sort of know but not be vocal about.
- For v4, we will probably want to have a look at our roadmap again, and update it. I am also happy to hear any ideas / suggestion of what (breaking) changes that will go into a v4 release.

**Comment by BenJamesBen** [comment, marked as answer]
### October 26, 2024 Community Call

#### Notes:
- TangerineShark: Is now back from Chicago.
- Wouter.vdug: Making a jacket. Making a lining for it. Linings are bigger than the rest of the garment.
- TangerineShark: Clothing to wear at work while acknowledging Halloween.
- Curvalicia: Bought Black fabric for a circle skirt. But, the fabric has holes in it, so it needs to be lined. How to hem the skirt and lining? Attach only at waistband because hems will stretch and drop and cause puckers. Will a double-circle skirt be too poofy? 75% or 50% circle skirt linings can also work and not be poofy. Side seam pockets.
- RockerKitten: Moving an office to a different room in the house.
- Vili: Now has a borrowed sewing machine and is looking for a project to make with it.
- Travel to increase web of trust.
- Docusaurus design docs porting
  - 24 unclaimed, 21 claimed, 15 done. We're on track.
  - Some folks have claimed designs that they want to make sure they are the ones to do them.
  - The process to convert to Docusaurus:
    1. Copy old design directory from old location to new location.
    2. Rename `en.md` to `readme.mdx` files.
    3. Edit the top-level, options, and instructions `readme.mdx` files to add needed frontmatter.
    4. Edit top-level `readme.mdx` and include all the text from Needs, Notes, Fabric, Cutting instructions.
    5. Edit the top-level and instructions `readme.mdx` files to format tip, note, info, warning, etc. custom tags into Adminitions. Also change URLs as needed for the next docs structure.
    6. Edit the `options/readme.mdx` file and manually construct its contents based on all the options for that design, putting the designs into the same order you want them to appear in the end doc. This is the most time-consuming part of the process.
    7. Edit all the individual `options/*/readme.mdx` files to convert things to admonitions.
    8. Delete the Needs, Notes, Fabric, and Cutting directory. Delete the Measurements directory (nothing needed from there).
    9. Delete all the `*.md` files.
    10. Leave all the old docs files in the old directory. You don't have to delete them.
  - See Aaron and Brian for examples for the frontmatter to put in the readme.mdx files.
- RockerKitten: Making pin-on buttons.
- Bobgeorgethe3rd: Making a walkaway dress.

#### Attendees:
- Benjamin F.
- bobgeorgethe3rd
- Curvalicia (a.k.a. Anna Puk)
- RockerKitten
- TangerineShark
- Vili
- Wouter.vdub
